app/mailers/notification_mailer.rb

```diff
@@ -72,12 +72,12 @@ def application_created(event, receiver)
   def account_created(event, receiver)
     @provider_account = event.provider
     @account          = event.account
-    @receiver         = receiver
-    @user             = event.user
+    @receiver         = receiver.decorate
+    @user             = event.user.decorate
     @event            = event
 
     mail to: @receiver.email,
-         subject: "#{@user.informal_name} from #{@account.name} signed up"
+         subject: "#{@user.decorate.informal_name} from #{@account.name} signed up"
```

Comment:
It should be decorated already, no?

```suggestion
         subject: "#{@user.informal_name} from #{@account.name} signed up"
```

Reply:
Right, my bad 😄